sql/sqlserver/server.go

```diff
@@ -13,7 +13,8 @@
 // permissions and limitations under the License. See the AUTHORS file
 // for names of contributors.
 //
-// Author: Vivek Menezes (vivek@cockroachlabs.com)
+// Author: Peter Mattis (peter@cockroachlabs.com)
+//         Vivek Menezes (vivek@cockroachlabs.com)
```

Comment:
I believe when we have multiple author lines, each one has the `Author: ` prefix.